COMPILING.md

```diff
@@ -1,3 +1,11 @@
+# BEFORE COMPILATION
```

Comment:
I'd say this should be moved down to the section on jbmc, and the instructions to do `make -C jbmc/src java-models-library-download` should be removed.